src/schema.jl

```diff
@@ -53,6 +53,29 @@ Base.merge!(a::Schema, b::Schema) = (merge!(a.schema, b.schema); a)
 Base.keys(schema::Schema) = keys(schema.schema)
 Base.haskey(schema::Schema, key) = haskey(schema.schema, key)
 
+function ==(first::Schema, second::Schema)
+    first === second && return true
+    first.schema === second.schema && return true
+    length(first.schema) != length(second.schema) && return false
+    for key in keys(first)
+        !haskey(second, key) && return false
+        get(second, key, nothing) != get(first, key, nothing) && return false
```

Comment:
I don't think `get` is necessary here since you've already checked that the key is present on both sides.
```suggestion
        second[key] != first[key] && return false
```